playground/tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest_container
+
+TW = pytest_container.Container(url="containers-storage:localhost/playground",forwarded_ports=[pytest_container.PortForwarding(container_port=8001)])
```

Comment:
Why are you relying on a locally built container and not just adding the containrefile here? Assuming that you are using `.devcontainer/Contairefile`, you could achieve this as follows :crossed_fingers: :
```suggestion
TW = pytest_container.DerivedContainer(
    base=" registry.access.redhat.com/ubi9/python-311:1-52",
    containerfile="""
USER root

COPY playground/tests/requirements.txt .

RUN pip3 install -r requirements.txt && \
    dnf install -y podman buildah && \
    dnf clean all
""",
    forwarded_ports=[pytest_container.PortForwarding(container_port=8001)]
)
```

Reply:
This is the right Containerfile:- https://github.com/redhat-et/locallm/blob/main/playground/Containerfile
Duplicating the Containerfile code in two source files is not something I'd like to do.

Reply:
Sure, but now your tests have an additional (undocumented) prerequisite. Also, nothing prevents you from reading the file in your python code.

Reply:
Additional undocumented prerequisite?

Reply:
Your tests will fail unless the user ran `buildah bud -f Containerfile -t playground .` in the root directory beforehand. If you give pytest_container the containerfile, then it'll build it yourself and the user needn't make this additional step themselves

Reply:
I would prefer the build is taken care by github actions job before pytest is executed

Reply:
@dcermak The desired workflow (not working yet) https://github.com/redhat-et/locallm/actions/runs/8423835317/job/23066387994?pr=96

Reply:
Probably caused by this: https://github.com/redhat-et/locallm/pull/96#discussion_r1538031367